applications/luci-app-usteer/htdocs/luci-static/resources/view/usteer/usteer.js

```diff
@@ -239,34 +248,48 @@ return view.extend({
 	}),
 	load: function() {
 		return Promise.all([
-			this.callHostHints(),
-			this.callGetRemotehosts(),
-			this.callGetRemoteinfo(),
-			this.callGetLocalinfo(),
-			this.callGetClients()
+			rpc.list('usteer'),
+			this.callHostHints().catch (function (){return null;},
+			this.callGetRemotehosts().catch (function (){return null;}),
+			this.callGetRemoteinfo().catch (function (){return null;}),
+			this.callGetLocalinfo().catch (function (){return null;}),
+			this.callGetClients().catch (function (){return null;})
 		]);
 	},
+
+
+	
 	render: function(data) {
 		var m, s, o;
-		Hosts = data[0];
-		Remotehosts=data[1];
-		Remoteinfo=data[2];
-		Localinfo=data[3];
-		Clients=data[4];
+
+		if (!('usteer' in data[0])) {
+			m = new form.Map('usteer', _('Usteer'),_('usteer not running. make sure it is installed and running. Try /etc/init.d/usteer start'));
```

Comment:
So ouor translators don't hate us, change to e.g.
```js
m = new form.Map('usteer', _('Usteer'),_('usteer not running. make sure it is installed and running. Try %s').format('<code>/etc/init.d/usteer start</code>');
```

Reply:
ok